content/english/data_types/protein_data/data.md

```diff
@@ -10,7 +10,7 @@ menu:
 
 ## Published Data
 
-The [Human Protein Atlas](https://www.proteinatlas.org) provides open access to protein expression data across human organs, tissues and cells. This protein resource is also organized into special sections focussing on the tissue expression, subcellular expression, proteins in the brain or in blood, proteins related to pathology and metabolic pathways.
+The [Human Protein Atlas](https://www.proteinatlas.org) provides open access to protein expression data across human organs, tissues and cells. This protein resource is organized into focused sections for tissue expression, subcellular expression, proteins in the brain or in blood, proteins related to pathology and metabolic pathways.
```

Comment:
Learn something new every day! https://en.wiktionary.org/wiki/focussed 😅

Reply:
Wow, I had no idea! Luckily, this is a "prestige publication" so we are definitely going with "focused"

Reply:
```suggestion
The [Human Protein Atlas](https://www.proteinatlas.org) provides open access to protein expression data across human organs, tissues and cells. This protein resource is organized into focussed sections for tissue expression, subcellular expression, proteins in the brain or in blood, proteins related to pathology and metabolic pathways.
```

Just a demo for @pekrau and @Sebastian-D of a suggested change in a pull request.

No need to accept it 😉